app/main/menus.js

```diff
@@ -67,7 +65,7 @@ function optionAbout() {
 function optionCheckForUpdates() {
   return {
     label: t('Check for Updates…'),
-    click: () => checkNewUpdates(true),
+    click: () => checkForUpdates(),
```

Comment:
maybe it makes sense to add an option to check for updates on startup automatically (which is enabled by default). This way we make sure users have the latest and the greatest version

Reply:
I would like to first have a version with just manual updating to confirm that it works, and then enable automatic update checks in the next version.